cypress/integration/editor_spec.js

```diff
@@ -18,11 +18,22 @@ context(
     it("Show a modal to rename the project", () => {
       cy.get(".sc-eJwWfJ.hZFFnD").click();
 
-      cy.get("h2").should("contain.text", "Name Your Project");
+      cy.get('[data-testid="modal"]').contains("Name Your Project", { matchCase: false });
     });
 
-    it.skip("Modal should dismiss when cancel button clicked", () => {});
+    it ("Modal should dismiss when cancel button clicked", () => {
+    
+     cy.get(".sc-eJwWfJ.hZFFnD").click();
+     cy.contains('Cancel').click()
+     cy.get(".modal").should("not.exist");
```

Comment:
**Observation:** I noticed that you used the `[data-testid="modal"]` selector above. Is there a particular reason why? Does referencing by a different selector open us up to any potential inconsistencies is the future?